### PR TITLE
Some Python API doc fixes

### PR DESCRIPTION
**Pull request by dmatrix**
## What changes are proposed in this pull request?

Remove ">>>" prompts from all Python API examples

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ x] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ x] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Review by smurching — APPROVED** [review]
LGTM

File: mlflow/azureml/__init__.py [hunk 1]
```diff
@@ -85,35 +85,38 @@ def build_image(model_uri, workspace, image_name=None, model_name=None,
...
-    >>>                                 synchronous=True)
-    >>> # If your image build failed, you can access build logs at the following URI:
-    >>> print("Access the following URI for build logs: {}".format(azure_image.image_build_log_uri))
-    >>>
-    >>> # Deploy the image to Azure Container Instances (ACI) for real-time serving
-    >>> webservice_deployment_config = AciWebservice.deploy_configuration()
-    >>> webservice = Webservice.deploy_from_image(
-    >>>                    image=azure_image, workspace=azure_workspace, name="<deployment-name>")
-    >>> webservice.wait_for_deployment()
+    .. code-block:: Python
```
Comment (harupy) [inline mlflow/azureml/__init__.py:88]: Python -> python?

Comment (dmatrix) [reply mlflow/azureml/__init__.py:88]: It's case insensitive. Either works okay. See the attached figure. 
But I can change it.

Screen Shot 2020-04-09 at 3 14 48 PM

Comment (dmatrix) [reply mlflow/azureml/__init__.py]: fixed.

File: mlflow/spark.py [hunk 2]
```diff
@@ -360,12 +363,15 @@ def save_model(spark_model, path, mlflow_model=Model(), conda_env=None,
...
-    >>> #your pyspark.ml.pipeline.PipelineModel type
-    >>> model = ...
-    >>> mlflow.spark.save_model(model, "spark-model")
+    .. code-block:: python
+        :caption: Example
+
+        from mlflow import spark
+        from pyspark.ml.pipeline.PipelineModel
+
+        #your pyspark.ml.pipeline.PipelineModel type
```
Comment (pogil) [inline mlflow/spark.py:372]: Add a space after the hash symbol?

**Review by pogil — APPROVED** [review]
Thanks @dmatrix! Just a couple of minor comments, otherwise looks good to me!

File: mlflow/spark.py (hunk 2 above)
Comment (dmatrix) [reply mlflow/spark.py:372]: Good catch. Will fix them and do a commit shortly

Comment (dmatrix) [reply mlflow/spark.py]: Fixed!

File: mlflow/mleap.py [hunk 3]
```diff
@@ -46,36 +46,38 @@ def log_model(spark_model, sample_input, artifact_path, registered_model_name=No
...
+        # Create an MLlib pipeline
+        tokenizer = Tokenizer(inputCol="text", outputCol="words")
+        hashingTF = HashingTF(inputCol=tokenizer.getOutputCol(), outputCol="features")
+        lr = LogisticRegression(maxIter=10, regParam=0.001)
+        pipeline = Pipeline(stages=[tokenizer, hashingTF, lr])
+        model = pipeline.fit(training)
+        # log parameters
+        mlflow.log_param("max_iter", 10)
+        mlflow.log_param("reg_param", 0.001)
+        #l og the Spark MLlib model in MLeap format
```
[the remark this answers is not in the document]
Comment (dmatrix) [reply mlflow/mleap.py]: fixed

File: mlflow/pytorch/__init__.py [hunk 4]
```diff
@@ -323,14 +329,17 @@ def load_model(model_uri, **kwargs):
...
-    >>> y_pred = pytorch_model(x_new_data)
+    .. code-block:: python
+        :caption: Example
+
+        import torch
+        import mlflow
+        import mlflow.pytorch
+        # Set values
+        model_path_dir = ...
+        run_id="96771d893a5e46159d9f3b49bf9013e2"
```
Comment (harupy) [inline mlflow/pytorch/__init__.py:340]:
```suggestion
        run_id = "96771d893a5e46159d9f3b49bf9013e2"
```

Comment (harupy) [reply mlflow/pytorch/__init__.py:340]: Add spaces around `=`?

Comment (dmatrix) [reply mlflow/pytorch/__init__.py:340]: I suppose to be consistent, we should.

Comment (dmatrix) [reply mlflow/pytorch/__init__.py]: fixed!

File: mlflow/gluon.py [hunk 5]
```diff
@@ -187,25 +193,28 @@ def log_model(gluon_model, artifact_path, conda_env=None):
...
+        net = HybridSequential()
+        with net.name_scope():
+            ...
+        net.hybridize()
+        net.collect_params().initialize()
+        softmax_loss = SoftmaxCrossEntropyLoss()
+        trainer = Trainer(net.collect_params())
+        est = estimator.Estimator(net=net, loss=softmax_loss, metrics=Accuracy(), trainer=trainer)
+        # Log metrics and log the model
+        with mlflow.start_run() as run:
```
Comment (harupy) [inline mlflow/gluon.py:215]:
```suggestion
        with mlflow.start_run():
```

Remove `as run` because `run` is not used?

Comment (dmatrix) [reply mlflow/gluon.py:215]: Sure, if you not going to using a variable in the context manager block, as any place elsewhere, you don't need it. 

Will remove it.

Comment (dmatrix) [reply mlflow/gluon.py]: fixed!

**Pull request merged**